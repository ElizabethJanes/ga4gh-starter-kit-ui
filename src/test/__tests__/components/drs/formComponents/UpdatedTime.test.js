import React from 'react';
import renderer from 'react-test-renderer';
import UpdatedTime from '../../../../../lib/components/drs/formComponents/UpdatedTime';

test('SHOW <UpdatedTime /> ', () => {
    const updatedTime = renderer
    .create(<UpdatedTime readOnly={true}
    updated_time='2021-07-22T13:00:00Z' />)
    .toJSON();
    //expect(updatedTime).toMatchSnapshot();
});

test('NEW and EDIT <UpdatedTime />', () => {
    const updatedTime = renderer
    .create(<UpdatedTime readOnly={false}
    updated_time='2021-07-22T13:00:00Z' />)
    .toJSON();
    //expect(updatedTime).toMatchSnapshot();
});