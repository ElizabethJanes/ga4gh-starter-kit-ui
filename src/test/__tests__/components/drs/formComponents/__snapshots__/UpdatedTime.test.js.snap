// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NEW and EDIT <UpdatedTime /> 1`] = `
<div
  class="MuiFormControl-root MuiFormControl-fullWidth"
>
  <div
    class="MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal"
    format="yyyy-MM-dd HH:mm:ss OOOO"
  >
    <label
      class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiFormLabel-filled"
      data-shrink="true"
      for="updated_time"
      id="updated_time-label"
    >
      Updated Time
    </label>
    <div
      class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
    >
      <input
        aria-describedby="updated_time-helper-text"
        aria-invalid="false"
        class="MuiInputBase-input MuiInput-input"
        id="updated_time"
        name="updated_time"
        readonly=""
        type="text"
        value="2021-07-22 09:00:00 GMT-04:00"
      />
    </div>
    <p
      class="MuiFormHelperText-root MuiFormHelperText-filled"
      id="updated_time-helper-text"
    >
      Timestamp of when the DRS Object was most recently updated in ISO 8601 format.
    </p>
  </div>
</div>
`;

exports[`SHOW <UpdatedTime />  1`] = `
<div
  class="MuiFormControl-root MuiFormControl-fullWidth"
>
  <div
    class="MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal"
    format="yyyy-MM-dd HH:mm:ss OOOO"
    readonly=""
  >
    <label
      class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiFormLabel-filled"
      data-shrink="true"
      for="updated_time"
      id="updated_time-label"
    >
      Updated Time
    </label>
    <div
      class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
    >
      <input
        aria-describedby="updated_time-helper-text"
        aria-invalid="false"
        class="MuiInputBase-input MuiInput-input"
        id="updated_time"
        name="updated_time"
        readonly=""
        type="text"
        value="2021-07-22 09:00:00 GMT-04:00"
      />
    </div>
    <p
      class="MuiFormHelperText-root MuiFormHelperText-filled"
      id="updated_time-helper-text"
    >
      Timestamp of when the DRS Object was most recently updated in ISO 8601 format.
    </p>
  </div>
</div>
`;
